configs/azure_iot_build_rules.cmake

```diff
@@ -74,7 +74,7 @@ elseif(UNIX) #LINUX OR APPLE
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror")
     if(NOT (IN_OPENWRT OR APPLE))
-        set (CMAKE_C_FLAGS "-D_POSIX_C_SOURCE=200112L ${CMAKE_C_FLAGS}")
+        set (CMAKE_C_FLAGS "-D_POSIX_C_SOURCE=200112L -D_XOPEN_SOURCE=500 ${CMAKE_C_FLAGS}")
```

Comment:
Sorry, could you add this comment on top please?
// _XOPEN_SOURCE=500 is required for glibc to expose random and srandom.

Reply:
Added comment! Thanks for the review @ewertons!